presto-docs/src/main/sphinx/sql/set-role.rst

```diff
@@ -0,0 +1,41 @@
+========
+SET ROLE
+========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SET ROLE ( role | ALL | NONE ) [ IN catalog ]
+
+Description
+-----------
+
+``SET ROLE`` sets the enabled roles for the current session in ``catalog``
+or in the current catalog if ``catalog`` is not specified.
+
+``SET ROLE role`` enables a single specified role for the current session.
+
+For ``SET ROLE role`` statement to succeed, the user executing it should be
```

Comment:
For the ``SET ROLE role`` statement to succeed, the user executing it should have a grant for the given role.